server/src/models/Review.js

```diff
@@ -13,7 +13,8 @@ class Review extends Model {
                 userId: {type: ["integer", "string"] },
                 shoeId: {type: ["integer", "string"] },
                 score: {type: ["integer", "string"], minimum:1, maximum:5 },
-                body: {type: "string"}
+                body: {type: "string"},
+                votes: { type: ["integer", "string"] }
```

Comment:
seeing plural "votes" should indicate to us that we need to normalize the data. how can one column store multiple values? not easily! which is why Votes should be it's own table